src/Raven.Server/Documents/Replication/ReplicationLoader.cs

```diff
@@ -402,14 +412,14 @@ private void HandleTopologyChange(DatabaseRecord newRecord, out List<OutgoingRep
             lock (_locker)
             {
                 instancesToDispose = new List<OutgoingReplicationHandler>();
-                if (newRecord == null)
+                if (newRecord == null || DatabaseTopology.PartOfCluster == false)
```

Comment:
Being part of a cluster is a global parameter, that affect every DB in the server.